ember-flight-icons/README.md

```diff
@@ -1,43 +1,44 @@
-ember-flight-icons
-==============================================================================
+# ember-flight-icons
 
-An Ember addon for the Flight Icons. Currently in development.
+An Ember addon for the icons from `flight`. Currently in development.
 
 Goals:
 
-* accessible
-* easy to use
-* easy to maintain
-* sensible defaults
+* Accessible
+* Easy to use
+* Easy to maintain
+* Sensible defaults
 
-Compatibility
-------------------------------------------------------------------------------
+## Compatibility
 
 * Ember.js v3.20 or above
 * Ember CLI v3.20 or above
 * Node.js v10 or above
 
-Installation
-------------------------------------------------------------------------------
+## Installation
 
 This package has not yet been published. Once it has, you can install it like any other Ember addon:
 
 ```bash
 ember install @hashicorp/ember-flight-icons
 ```
 
-Usage
-------------------------------------------------------------------------------
-The `name` must be specified.
+## Usage
+
+### Required usage
+
+The `name` must be specified. A default icon size of 16 will be provided.
```

Comment:
```suggestion
The `name` must be specified. A default icon size of 24 will be provided.
```
24x24 is the default size.

Reply:
oh gotcha, i assumed it was the other way around. any reason why? curious if more icons are 16 vs 24 rn. but, let's address that separately, i'll make a triage issue